### PR TITLE
Add support for Jablotron tags

### PR DESCRIPTION
**Pull request by mauimauer**
# What's new

New protocol for lfrfid
- Jablotron tags(125KHz, ASK, Manchester Biphase, 64bit)

# Verification 

- Use the LF-RFID application as usual, try to read, emulate, write new protocol.

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: lib/lfrfid/protocols/protocol_jablotron.c [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        // check 11 bits preamble
+        if(bit_lib_get_bits_16(protocol->encoded_data, 0, 16) != 0b1111111111111111) break;
+        // check next 11 bits preamble
+        if(bit_lib_get_bits_16(protocol->encoded_data, 64, 16) != 0b1111111111111111) break;
+
+        uint8_t checksum = bit_lib_get_bits(protocol->encoded_data, 56, 8);
+        if(checksum != protocol_jablotron_checksum(protocol->encoded_data)) break;
+
+        result = true;
+    } while(false);
```
Comment (miili) [inline lib/lfrfid/protocols/protocol_jablotron.c]: eliminate while loop, `return false` for the breaking conditions. Improves readability.

**Review by miili — COMMENTED** [review]
Can you add a link to the whitepaper / data sheet?

**Comment by mauimauer** [comment]
> Can you add a link to the whitepaper / data sheet?

https://github.com/Proxmark/proxmark3/blob/master/client/cmdlfjablotron.c is basically the only reference I could find.
This implementatiopn has been tested with proxmark's sim and reader for Jablotron

File: lib/lfrfid/protocols/protocol_jablotron.c (hunk 1 above)
Comment (mauimauer) [reply lib/lfrfid/protocols/protocol_jablotron.c]: Fair.

**Comment by kubicek** [comment]
I have tested reading and emulating and it works! Thank you!

**Review by DrZlo13 — APPROVED** [review]

**Pull request merged**